tests/test_utils.py

```diff
@@ -216,3 +217,21 @@ def test_is_valid():
     """Check the function is_valid()."""
     assert not is_valid(None)
     assert is_valid("parameter")
+
+
+def test_get_core_dependencies():
+    """Check the function get_core_dependencies()."""
+    result = [
```

Comment:
@sunilk747  Do Not hard-code the values assert for the data exist in the response or not